tmol/score/common/uaid_util.hh

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+
+#include <tmol/utility/tensor/TensorAccessor.h>
+#include <tmol/utility/tensor/TensorPack.h>
+#include <tmol/utility/tensor/TensorStruct.h>
+#include <tmol/utility/tensor/TensorUtil.h>
+
+#include <tmol/score/common/diamond_macros.hh>
+
+namespace tmol {
+namespace score {
+namespace common {
+
+template <typename Real, int N>
+using Vec = Eigen::Matrix<Real, N, 1>;
+
+template <typename Real, typename Int, tmol::Device D>
+TMOL_DEVICE_FUNC int resolve_atom_from_uaid(
+    TensorAccessor<Int, 1, D> uaid,
+    int block_index,
+    int pose_index,
+
+    TView<Int, 2, D> pose_stack_block_coord_offset,
+    TView<Int, 2, D> pose_stack_block_type,
+    TView<Vec<Int, 2>, 3, D> pose_stack_inter_block_connections,
+    TView<Int, 3, D> block_type_omega_quad_uaids,
```

Comment:
@aleaverfay I just noticed while searching for omega that this rogue argument made its way into this function. What's the best way to remove this? Should I make a mini-PR? Or do you just want to update this when you use this function for your task that needed it?